environment.h

```diff
@@ -28,36 +28,27 @@ static std::map<std::string, std::function<double(std::vector<double>)>> string_
 struct env_param
 {
     NLOHMANN_DEFINE_TYPE_INTRUSIVE(env_param,
-                                   targetA,
-                                   targetB)
-double targetA;
-double targetB;
+                                   dummy)
```

Comment:
Yeah for now we cannot save anything. We could just delete and add it back when we need it.